docs/concepts/clickhouse.rst

```diff
@@ -0,0 +1,24 @@
+Clickhouse
```

Comment:
Can you check that the capitalization is "ClickHouse" (and "Superset" etc) everywhere?